polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -303,9 +309,9 @@ class CodeCell extends Cell {
 
     private errorMarkers: ErrorMarker[] = []
 
-    constructor(dispatcher: NotebookMessageDispatcher, notebookState: NotebookStateHandler, cellState: StateHandler<CellState>, private path: string) {
-        super(dispatcher, notebookState, cellState);
-
+    constructor(dispatcher: NotebookMessageDispatcher, notebookState: NotebookStateHandler, cell: StateHandler<CellState>, private path: string) {
+        super(dispatcher, notebookState, cell);
+        const cellState = this.cellState;
```

Comment:
might as well get rid of this `const` too?

Reply:
OK. Once it won't cause merge conflicts, will do